crates/core/app/src/action_handler/transaction.rs

```diff
@@ -179,11 +179,9 @@ mod tests {
         };
         let mut rng = OsRng;
         let tx = plan
-            .build_concurrent(&mut rng, fvk, witness_data)
+            .build_concurrent(fvk, &witness_data, &auth_data)
             .await
-            .expect("can build transaction")
-            .authorize(&mut rng, &auth_data)
-            .expect("can authorize transaction");
+            .expect("can't build transaction");
```

Comment:
Note: this should be `.expect("can build transaction")`, since the convention for `.expect` is that the string names the expected invariant.